app/assets/javascripts/components/overview/salesforce_link.jsx

```diff
@@ -10,6 +10,12 @@ const SalesforceLink = createReactClass({
     current_user: PropTypes.object
   },
 
+  updateSalesforceId() {
+    const rawSalesforceId = prompt('Update the Salesforce record ID or url for this course.');
+    const salesforceId = extractSalesforceId(rawSalesforceId);
+    ServerActions.linkToSalesforce(this.props.course.id, salesforceId);
+  },
+
```

Comment:
Hmm... I think other wording works just fine for updating as well. `linkToSalesforce` could just be reused with the different button text and I think it'll be fine.

Reply:
Ah okay! Will do.